showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITHttpAnnotation.java

```diff
@@ -39,38 +39,33 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
 import java.util.stream.Collectors;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
 // This test runs from the parameters in the compliance_suite.json file
 // The file is downloaded from the gapic-showcase repo. Each compliance
 // group is a set of HttpJson behaviors we want to test for. Each group
 // tests the product of the rpc list and requests list.
-@RunWith(Parameterized.class)
-public class ITHttpAnnotation {
+class ITHttpAnnotation {
 
-  @Parameterized.Parameters(name = "Compliance Group Name: {0}")
-  public static String[] data() {
-    return new String[] {
-      "Fully working conversions, resources",
-      "Binding selection testing",
-      "Cases that apply to non-path requests",
-      "Fully working conversions, no resources"
-    };
+  public static Stream<Arguments> data() {
```

Comment:
nit: Probably doesn't matter, but to keep it consistent perhaps this can be private?